server/lib/nl/fulfillment/utils.py

```diff
@@ -165,6 +166,26 @@ def get_default_contained_in_place(places: List[Place],
   return constants.DEFAULT_PARENT_PLACES.get(ptype, None)
 
 
+# Get facet id to use when there is a date specified (use a facet id that has
+# data). Gets the facet id that has data for the most places.
+def get_facet_id(sv: str, date: Date, sv_exist_facet: Dict[str, Dict[str, str]],
```

Comment:
Does the new structure of sv_exist_facet impact is_coplottable?

Reply:
good call, updated!